.tests/compatibility/reference/WASH7P.fna.fai

```diff
@@ -0,0 +1 @@
+chr1	15009	68	70	71
```

Comment:
same

Reply:
this one we actually need=) It's used to decide fetch which chromosome will be used to merge files

Reply:
true